src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -67,6 +68,11 @@ internal class SourceLocalSymbol : LocalSymbol
             _locations = ImmutableArray.Create<Location>(identifierToken.GetLocation());
         }
 
+        internal Binder Binder
```

Comment:
Scope is owned by a Binder, each local knows this Binder (there is no change here), now consumers of the local can get to that Binder (that is new).